### PR TITLE
second round of suggested changes

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: src/Microsoft.Identity.Web/DownstreamWebApiSupport/DownstreamWebApi.cs [hunk 1]
```diff
@@ -113,11 +110,7 @@ public class DownstreamWebApi : IDownstreamWebApi
                 user,
                 new StringContent(JsonSerializer.Serialize(input), Encoding.UTF8, "application/json")).ConfigureAwait(false);
 
-            try
-            {
-                response.EnsureSuccessStatusCode();
```
Comment (jmprieur) [inline src/Microsoft.Identity.Web/DownstreamWebApiSupport/DownstreamWebApi.cs:118]: Can't response.EnsureSuccessStatusCode throw?
it seems it does: 
https://stackoverflow.com/questions/21097730/usage-of-ensuresuccessstatuscode-and-handling-of-httprequestexception-it-throws

File: src/Microsoft.Identity.Web/Resource/RequiredScopeFilter.cs [hunk 2]
```diff
@@ -50,7 +50,7 @@ public void OnAuthorization(AuthorizationFilterContext context)
 
         private void ValidateEffectiveScopes(AuthorizationFilterContext context)
         {
-            if (_effectiveAcceptedScopes == null || !_effectiveAcceptedScopes.Any())
+            if (_effectiveAcceptedScopes == null || _effectiveAcceptedScopes.Length < 0)
```
Comment (jmprieur) [inline src/Microsoft.Identity.Web/Resource/RequiredScopeFilter.cs:53]: probably == 0

**Review by jmprieur — COMMENTED** [review]
I've 2 questions.

**Review by jmprieur — APPROVED** [review]
LGTM
Thanks @jennyf19

File: src/Microsoft.Identity.Web/DownstreamWebApiSupport/DownstreamWebApi.cs (hunk 1 above)
Comment (jennyf19) [reply src/Microsoft.Identity.Web/DownstreamWebApiSupport/DownstreamWebApi.cs]: I reverted back to what we have for now. will take a more indepth look at this later

**Pull request merged**